### ISSUE TITLE
Add LearningRateScheduler for adam

### ISSUE THREAD

**Issue by djanloo** [djanloo/cmepda#7]
It seems that the model can get better but the learning rate has to be adjusted during training.

Tested Adam with lr=0.1 and it's utterly slow
Tested Adam with lr=0.0001 with the same results

Best option at the moment is lr=0.001 (default) but I think learning can be sped up:
![learning_rate](https://user-images.githubusercontent.com/89815653/177216475-9d79a8e1-1fe7-4b63-baf1-8fb038765b44.png)

**Comment by luciapapalini** [comment]
Maybe we can evaluate to implement "curriculum learning" (?) or something like that

**Comment by djanloo** [comment]
Linearly increasing lr : 0.01 -> 0.4 (step = 0.02). Awful.
![linearly_increasing_lr](https://user-images.githubusercontent.com/89815653/177377583-cf5ddde7-a2ac-427a-a9a8-73931b564a1c.png)

**Comment by luciapapalini** [comment]
We try with curriculum learning first
